### DISCUSSION TITLE
meta workshop intro.md and PuTTY

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#671] [General]
I discovered that PuTTY does *not* use the same ssh keys that the `ssh` command uses - it has its own. You can convert the `ssh public key` to a PuTTY key however.

Should the `intro.md` make note of that?

**Comment by lbdyck** [comment]
Using Microsofts CoPilot I got this which may help:

[PuTTY and OpenSSH use different formats for public SSH keys](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[1](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[2](https://www.ionos.ca/digitalguide/server/configuration/ssh-key-with-putty/). [PuTTY uses its own .ppk format, while OpenSSH uses a PEM format](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[1](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key). [You can use PuTTYgen to convert between these formats if you need to use both clients](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[1](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[2](https://www.ionos.ca/digitalguide/server/configuration/ssh-key-with-putty/).

[If you generate a key with OpenSSH using ssh-keygen with the default options, it will work with most SSH servers](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[1](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key). [However, if you generate a key with PuTTYgen, you may need to adjust some parameters, such as the number of bits or the algorithm, to make it compatible with the server](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key)[1](https://unix.stackexchange.com/questions/74545/what-is-the-difference-between-an-openssh-key-and-putty-key).

**Reply by IgorTodorovskiIBM** [reply]
FYI @HarithaIBM

**Reply by HarithaIBM** [reply]
Oh Ok!
